test/manipulation.test.js

```diff
@@ -768,6 +768,86 @@ describe('manipulation', function() {
     });
   });
 
+  describe('upsertWithWhere', function() {
+    var ds = getSchema();
+    var Post;
+    before('prepare "Post" model', function(done) {
+      Post = ds.define('Post', {
+        empid: { type: Number, id: true },
```

Comment:
I find this test model confusing. What is `empid` and how is it related to `Post`? Why cannot we use `id` instead?